docs-beta/src/components/Features.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { styles } from "./shared-styles"
+import "../css/tailwind-styles.css"; 

```

Comment:
You shouldn't need to import the css file I wouldn't think? (this goes for all the component files :))